app/controllers/spaces_controller.rb

```diff
@@ -42,7 +42,7 @@ def create
   def update
     if space.update(space_params)
       flash[:notice] = t(".success")
-      redirect_to space
+      redirect_to space.location(:edit), allow_other_host: true
```

Comment:
Since the `Space#edit` page is so big; it felt weird to be sending them back to the `Space#show` page on change. Also we didn't have any tests about where it was supposed to go which implied to me that it was left unspecified.